### PR TITLE
added new lines 1 & 2

### PR DESCRIPTION
**Pull request by asgtheone**
as requested added two new lines

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,3 +2,7 @@
 Test repository
 
 My first git commit
+
+new line 1
+
```
Comment (spremathilaka) [inline README.md:7]: please remove extra line here

Comment (asgtheone) [reply README.md:7]: these extra lines added for more clarity

Comment (spremathilaka) [reply README.md:7]: okey then please meake the first letter caps

Comment (asgtheone) [reply README.md:7]: ok I'll do it now

**Review by spremathilaka — APPROVED** [review]
well done Amila 👍

**Review by spremathilaka — APPROVED** [review]

**Pull request merged**